Adapter/ImageMediaAdapter.php

```diff
@@ -132,8 +132,14 @@ private function updateMedia(MediaInterface $media, UploadedFile $uploadedFile,
             $latestFileVersion = $existingFile->getLatestFileVersion();
 
             if ($latestFileVersion && $latestFileVersion->getSize() === $uploadedFile->getSize()) {
-                // same image, not necessary to update anything
-                return $media;
+                try {
+                    $this->storage->load($latestFileVersion->getStorageOptions());
+
+                    // same image, not necessary to update anything
+                    return $media;
+                } catch (\Exception $e) {
```

Comment:
@Prokyonn is there a specific exception class?

Reply:
for the `LocalStorage` it's just an [ErrorException](https://www.php.net/manual/en/class.errorexception.php), but I don't know what exception other adapters throw

Reply:
OK then its ok